packages/tasks/src/model-libraries.ts

```diff
@@ -590,6 +590,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 			OR path:"open_clip_pytorch_model.bin"
 			OR path:"pytorch_model.bin"`,
 	},
+	"open-sora": {
+		prettyLabel: "OpenSora",
+		repoName: "OpenSora",
+		repoUrl: "https://github.com/hpcaitech/Open-Sora",
+		filter: false,
+		countDownloads: `path:"model.safetensors"`,
```

Comment:
As of now, https://huggingface.co/hpcai-tech/Open-Sora-v2 don't have any `model.safetensors` file. Did you mean

```suggestion
		countDownloads: `path:"Open_Sora_v2.safetensors"`,
```

?